textinput/textinput.go

```diff
@@ -203,6 +203,11 @@ func (m Model) Cursor() int {
 	return m.pos
 }
 
+// Blink returns whether the cursor should blink or not.
```

Comment:
I'd reword this to “Blink returns whether or not to draw the cursor,” just to disambiguate it from cursor states like `CursorStatic`, `CursorBlink` and `CursorHide`.

Reply:
Thanks, I agree that sounds much better. Fixed now.